### PR TITLE
Uses correct conversion from kcal/mol -> kj/mol

### PR DESCRIPTION
**Pull request by badisa**

### CODE REVIEW

File: timemachine/fe/utils.py [hunk 1]
```diff
@@ -47,7 +49,7 @@ def convert_uIC50_to_kJ_per_mole(amount_in_uM):
     """
     TODO: more sig figs
     """
-    return 0.593 * np.log(amount_in_uM * 1e-6) * 4.18
+    return 0.593 * np.log(amount_in_uM * 1e-6) * constants.KCAL_TO_KJ
```
Comment (badisa) [inline timemachine/fe/utils.py:52]: @proteneer Do you know where the `0.593` comes from?

File: timemachine/fe/utils.py [hunk 2]
```diff
@@ -65,7 +67,7 @@ def convert_uM_to_kJ_per_mole(amount_in_uM):
         Binding potency in kJ/mol.
 
     """
-    return 0.593 * np.log(amount_in_uM * 1e-6) * 4.18
+    return 0.593 * np.log(amount_in_uM * 1e-6) * constants.KCAL_TO_KJ
```
Comment (maxentile) [inline timemachine/fe/utils.py:70]: Based on prior notes, I think 0.593 is intended to be `molar gas constant * temperature`, in units of kcal/mol, but
(1) the temperature may be inconsistent with `DEFAULT_TEMP = 300` kelvin
(2) there may be precision loss.

https://www.wolframalpha.com/input?i=molar+gas+constant+*+300+kelvin = 0.5962 kcal/mol
https://www.wolframalpha.com/input?i=molar+gas+constant+*+298+kelvin = 0.5922 kcal/mol

**Review by maxentile — APPROVED** [review]

File: timemachine/fe/utils.py (hunk 2 above)
Comment (proteneer) [reply timemachine/fe/utils.py:70]: `RT` is from the table

https://en.wikipedia.org/wiki/KT_(energy) (measured in kcal/mol)

Note that the `T` here should probably be consistent with experimental temperatures, not simulation temperatures, if converting labelled data.

Comment (badisa) [reply timemachine/fe/utils.py:70]: Should we parameterize this to take in an experimental temperature?

**Review by jkausrelay — APPROVED** [review]

File: timemachine/fe/utils.py (hunk 2 above)
Comment (proteneer) [reply timemachine/fe/utils.py:70]: probably but lets default it to the IUPAC convention of `298.15 Kelvin` 

https://sciencenotes.org/what-is-room-temperature/#:~:text=Room%20Temperature%20in%20Science%20and%20Industry&text=The%20International%20Union%20of%20Pure,%C2%B0F%2C%20298.15%20K).

Comment (badisa) [reply timemachine/fe/utils.py]: Done in [ee688cd](https://github.com/proteneer/timemachine/pull/825/commits/ee688cdb3bf110e59c9707fa9e6729df283a9d05) and [c23e492](https://github.com/proteneer/timemachine/pull/825/commits/c23e4926661325266498abfb64c2731bb9c384c8)

Comment (maxentile) [reply timemachine/fe/utils.py]: We don't have to change this now, but should project-wide `DEFAULT_TEMP` also be changed from 300 to 298.15?

**Review by maxentile — APPROVED** [review]
Nice changes and addition of `constants.MOLAR_GAS_CONSTANT`

File: timemachine/fe/utils.py (hunk 2 above)
Comment (maxentile) [reply timemachine/fe/utils.py]: (Irrelevant to current PR -- moved to Issue https://github.com/proteneer/timemachine/issues/827 for later reminder)

Comment (proteneer) [reply timemachine/fe/utils.py]: yes

**Review by proteneer — APPROVED** [review]
lgtm

**Pull request merged**